cdap-docs/admin-manual/source/_includes/installation/configuration.txt

```diff
@@ -48,6 +48,14 @@ for CDAP to run successfully.
      $ sudo cp -f /etc/cdap/conf.mycdap/cdap-site.xml.example /etc/cdap/conf.mycdap/cdap-site.xml
      $ sudo vi /etc/cdap/conf.mycdap/cdap-site.xml
 
+#. If necessary, configure a ``cdap-env.sh`` after you have installed the CDAP packages.
```

Comment:
"... customize cdap-env.sh...".  we include a cdap-env.sh with popular options commented out, and if they follow the docs they will have it already.